cli/cli/out/file_logger.go

```diff
@@ -0,0 +1,71 @@
+package out
+
+import (
+	"github.com/kurtosis-tech/kurtosis/cli/cli/helpers/host_machine_directories"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	"io/fs"
+	"os"
+)
+
+// FileLogger this logger will only log to a file
+// TODO: In commands like inspect we use out.PrintOutLn - will need to add this fileLogger
+//  to print commands' output as well.
+var FileLogger = logrus.New()
```

Comment:
I suggest you do not use a global var for saving the logger, you could modify the `SetupFileLogger` and convert it to a `GetFileLogger` which implements a singleton pattern and store the values in a private var.
You can use the same pattern that we use for `GetMetricsUserIDStore() *MetricsUserIDStore` you will need to create a new `FileLogger` struct

Reply:
good catch; will do that.